README.md

```diff
@@ -48,20 +48,21 @@ TBA
+| Name               | Default     | Description                                                                                                                                              |
+| ----------------   | -------     | ------------------------------------------------------------------------------------------------------------                                             |
+| groupOffset        |             |                                                                                                                                                          |
+| startDate          |             |                                                                                                                                                          |
+| endDate            |             |                                                                                                                                                          |
+| snapMinutes        |             |                                                                                                                                                          |
+| showCursorTime     |             |                                                                                                                                                          |
+| cursorTimeFormat   |             |                                                                                                                                                          |
+| itemHeight         |             |                                                                                                                                                          |
+| timelineMode       |             |                                                                                                                                                          |
+| timebarFormat      |             |                                                                                                                                                          |
+| itemRenderer       |             |                                                                                                                                                          |
+| groupRenderer      |             |                                                                                                                                                          |
+| shallowUpdateCheck | False       | If true timeline will try to minimize re-renders . Set to false if items don't show up/update on prop change                                             |
+| shallowUpdateFunc  | () => False | Function called when `shallowUpdateCheck`==true. If returns true the timeline will be redrawn. If false the library will decide if redrawing is required |
```

Comment:
since this is essentially a force redraw function, should it be named like 'requireRedrawFunc' or 'forceRenderFunc' or similar ?